symfony/client/src/core/util/validation/__tests__/rules.spec.ts

```diff
@@ -154,3 +159,60 @@ describe('core/util/validation/rules::endDateShouldBeAfterStartDate', () => {
     expect(result).toBe('End date should be after start date');
   });
 });
+
+describe('core/util/validation/rules::validPhoneNumberFormat', () => {
```

Comment:
@samanthajayasinghe any feedback on phone number validation for FE.
Currently BE don't have strict validation

Reply:
don't validate the phone number .. it's change county to country

Reply:
hi @samanthajayasinghe we only validatate  numbers and  + - / ( ) and check if it's less than 25, so it won't be problem right ? although there are different country codes and styles